db/helpers/botDataUtils.js

```diff
@@ -251,7 +251,7 @@ function updateValues(seq, instance) {
           });
 
           // Find bot data to update
-          promises.push(BotData.findOne(whereConst));
+          promises.push(BotData.findOne(whereConst).catch((err) => err));
```

Comment:
Could be useful to log out the error here just in case we run into issues here in the future

Reply:
`(err) => err` is a no-op... if that is intentional, it is useful to add a comment to indicate that it is intentional. Under what circumstances would this throw?